Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -9,6 +9,9 @@ import TuistSupport
 // MARK: - PackageInfo Mapper Errors
 
 enum PackageInfoMapperError: FatalError, Equatable {
+    /// Thrown when the default path folder is not present.
+    case cantFindDefaultPath(AbsolutePath, String)
```

Comment:
`defaultPathNotFound` sounds like a better naming - but it is subjective, so just a suggestion.